neuralmonkey/trainers/generic_trainer.py

```diff
@@ -40,6 +39,7 @@ class Objective(NamedTuple(
 
 
 # pylint: disable=too-few-public-methods,too-many-locals,too-many-branches
+# pylint: disable=too-many-statements
```

Comment:
too many statements znamená, že by se to mělo rozsekat na funkce. nejde to nějak?

Reply:
souhlasim, nechtelo se mi v tom vcera vrtat... kazdopadne to jeste v tomhle PR zkusim zapracovat